addons/mail/views/mail_template_views.xml

```diff
@@ -50,7 +50,7 @@
                         <notebook>
                             <page string="Content" name="content">
                                 <field name="can_write" invisible="1"/>
-                                <field name="body_html" widget="html_legacy" class="oe-bordered-editor"
+                                <field name="body_html" widget="html" class="oe-bordered-editor"
```

Comment:
you should use widget="html_mail". It s the new field for style inline html field